src/integrationTest/java/integration/tests/TimestampTest.java

```diff
@@ -131,6 +131,18 @@ void shouldRemoveOffsetDIffWhenTimestampOffsetHasChangedCET() throws SQLExceptio
 		}
 	}
 
+	@Test
+	void shouldHaveCorrectInfo() throws SQLException {
+		try (Connection connection = this.createConnection("system");
+			 Statement statement = connection.createStatement();
+			 ResultSet resultSet = statement.executeQuery("SELECT 3::decimal")) {
+			 resultSet.next();
+			 assertEquals(9, resultSet.getMetaData().getScale(1));
+			assertEquals(38, resultSet.getMetaData().getPrecision(1));
+		}
+
+	}
```

Comment:
I don't think this test belongs here. This class deals with testing the Timestamp types.

Reply:
Although this test exists since March I am moving to to special test case.